enterprise/api/v1/views.py

```diff
@@ -1249,3 +1250,68 @@ def post(self, request):
                 {'error': str('Notification read request failed')},
                 status=HTTP_500_INTERNAL_SERVER_ERROR
             )
+
+
+class EnterpriseCustomerReportTypesView(APIView):
+    """
+    API for getting the report types associated with an enterprise customer
+    """
+    authentication_classes = [SessionAuthentication]
+    permission_classes = [permissions.IsAuthenticated]
+    http_method_names = ['get']
```

Comment:
can we use [SAFE_METHODS](https://github.com/encode/django-rest-framework/blob/1fa5bc31c0c047eb2109929b17f8e53e84b7d40b/rest_framework/permissions.py#L8) like below?
```python
from rest_framework.permissions import SAFE_METHODS
http_method_names = SAFE_METHODS
```